Tasks/VSBuild/task.json

```diff
@@ -63,21 +63,24 @@
             "type": "string",
```

Comment:
There are no generated files here, was the build run? Version also has to be bumped.